src/StatusQ/Components/StatusChatToolBar.qml

```diff
@@ -93,15 +93,19 @@ Rectangle {
 
             property bool showMoreMenu: false
             onClicked: {
-                if (showMoreMenu) {
+                if (!showMoreMenu) {
                     popupMenuSlot.item.popup(-popupMenuSlot.item.width + menuButton.width, menuButton.height + 4)
+                    statusChatToolBar.menuButtonClicked()
                 }
-                highlighted = true
-                statusChatToolBar.menuButtonClicked()
             }
 
-            onPressed: {
-                showMoreMenu = !showMoreMenu;
+            Timer {
+                id: menuClosedUpdater
+                interval: 100
+                repeat: false
+                onTriggered: {
+                    menuButton.showMoreMenu = false
+                }
```

Comment:
Just for me to understand:

Why is this timer necessary?

Reply:
i dont understand reason but menu dont have "invisible clickable area" or outside event blocker. So focused on "double clicking menu button" if you try to push on button with opened menu, the menu will close before the button receives mouse clicks. Without timer its impossible process two cases: 
-  Menu closed by button click
-  Menu closed by outside click
Thats why i add this ugly solution.